Sources/ParseSwift/Types/ParseError.swift

```diff
@@ -421,11 +421,11 @@ public extension Error {
      Returns the respective `ParseError` if the given `ParseError` code is equal to the error.
      
     **Example use case:**
-    ~~~
+    ````
```

Comment:
Did you check to see how Xcode renders these when looking at the documentation?

Reply:
Yes, it works the same, but the official documentation about markup says to use this for code blocks, so I changed it.

Reply:
I just Build the documentation in Xcode, to see how it looks here. It are code blocks, but just not in color.